modules/webgl/src/adapter/resources/webgl-transform-feedback.ts

```diff
@@ -0,0 +1,253 @@
+import type {Device, ShaderLayout, TransformFeedbackProps} from '@luma.gl/core';
+import {log, isObjectEmpty, TransformFeedback, Buffer} from '@luma.gl/core';
+import {GL} from '@luma.gl/constants';
+import {WebGLDevice} from '../webgl-device';
+import {WEBGLBuffer, isWebGL2} from '../..';
+
+/** For bindRange */
+type BufferRange = {
+  buffer: WEBGLBuffer;
+  byteOffset?: number;
+  byteLength?: number;
+};
+
+export class WEBGLTransformFeedback extends TransformFeedback {
+  get [Symbol.toStringTag](): string {
+    return 'TransformFeedback';
```

Comment:
Should this return `WEBGLTransformFeedback`? Not 100% sure but should be consistent across various classes.